config/config.go

```diff
@@ -149,6 +156,23 @@ func FromFlags() (*Config, error) {
 			}
 			cfg.COAKeys[i] = pk
 		}
+	} else if cloudKMSKeys != "" {
+		kmsKeys := strings.Split(cloudKMSKeys, ",")
```

Comment:
validate other values that are needed are not empty

Reply:
Added in https://github.com/onflow/flow-evm-gateway/pull/282/commits/074ba63dc15418e30f97a7fb7b3294d70a20d236